static/homepage/Nov_2023/2023-11-20-homepage.json

```diff
@@ -0,0 +1,289 @@
+            "contributors": [
+                "Anthony Towns",
+                "Antoine Riard",
+                "David A. Harding",
+                "Erik Aronesty",
+                "Rusty Russell",
+                "ZmnSCPxj"
+            ],
+            "file_path": "static/lightning-dev/Sept_2023/004092_Scaling-Lightning-With-Simple-Covenants.xml",
+            "combined_summ_file_path": "static/lightning-dev/Sept_2023/combined_Scaling-Lightning-With-Simple-Covenants.xml"
+        }
+    ],
+    "random_flashback_posts": [
+        {
+            "id": "014802",
+            "title": "Updating the Scaling Roadmap [Update]",
+            "link": "https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-July/014802.html",
+            "authors": [
+                "Paul Sztorc"
+            ],
+            "published_at": "2017-07-17T17:13:30+00:00",
+            "summary": "- Paul Sztorc proposed an updated Core Scalability Roadmap, with a draft on GitHub.\n- Gregory Maxwell opposed the update and Tom Zander disagreed with its \"Bitcoin\" labeling.\n- The roadmap includes new technologies and suggests a potential block size increase via hard fork.",
+            "n_threads": 5,
+            "dev_name": "bitcoin-dev",
+            "contributors": [
+                "Alex Morcos",
+                "Peter Todd"
+            ],
+            "file_path": "static/bitcoin-dev/July_2017/014802_Updating-the-Scaling-Roadmap-Update-.xml",
+            "combined_summ_file_path": "static/bitcoin-dev/July_2017/combined_Updating-the-Scaling-Roadmap-Update-.xml"
+        },
+        {
+            "id": "001697",
+            "title": "CPFP Carve-Out for Fee-Prediction Issues in Contracting Applications (eg Lightning)",
+            "link": "https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-November/001697.html",
+            "authors": [
+                "Matt Corallo"
+            ],
+            "published_at": "2018-11-29T19:37:54+00:00",
+            "summary": "- Lightning transactions involve pre-signed future transactions, but complexity causes channel issues.\n- Counterparties can disrupt or increase costs of Lightning transactions via feerate manipulation.\n- Tweaking Lightning transactions or marking them for RBF could improve security and relay issues.",
+            "n_threads": 16,
+            "dev_name": "lightning-dev",
+            "contributors": [
+                "Bob McElrath",
+                "David A. Harding",
+                "Jeremy",
+                "Johan Tor\u00e5s Halseth",
+                "Rusty Russell"
+            ],
+            "file_path": "static/lightning-dev/Nov_2018/001697_CPFP-Carve-Out-for-Fee-Prediction-Issues-in-Contracting-Applications-eg-Lightning-.xml",
+            "combined_summ_file_path": "static/lightning-dev/Nov_2018/combined_CPFP-Carve-Out-for-Fee-Prediction-Issues-in-Contracting-Applications-eg-Lightning-.xml"
+        }
+    ],
+    "today_in_history_posts": []
```

Comment:
this remains empty. Can we just drop this and focus on one post for bitcoin and one for LN?